### PR TITLE
Migrated the site to Vanilla V1

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Migrated the site to Vanilla V1

## QA
- Pull down code
- Run `/run`
- Check the site looks ok 

## Issue / Card
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/5 

## Demo
http://cloud-init.io-v1.demo.haus/

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,18 +1,20 @@
-<!-- Footer -->
-<div class="wrapper">
-    <footer class="footer">
-
-        <p class="footer__text">&copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
-
-        <ul class="footer__links">
-            <li class="footer__list-item"><a href="http://www.ubuntu.com//legal">Legal information</a><span class="footer__dot"> &#183;</span></li>
-            <li class="footer__list-item"><a href="https://github.com/ubuntudesign/cloud-init/issues/new">Report a bug on this site</a></li>
+<footer class="p-footer" role="contentinfo">
```
Comment (anthonydillon) [inline _includes/footer.html]: Don't need `role="contentinfo"`

File: _sass/main.scss [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
-'footer',
-'hero',
-'media-unit',
-'media-box';
+// Optionally override some settings
+$color-brand: #e95420;
+$color-accent: #2c001e;
+
+// Import the Vanilla Brochure Theme - https://github.com/vanilla-framework/vanilla-brochure-theme
+@import "../node_modules/vanilla-brochure-theme/scss/theme";
```
Comment (anthonydillon) [inline _sass/main.scss]: We should use an includePath when creating the CSS

File: _sass/main.scss [hunk 3]
```diff
@@ -1,18 +1,39 @@
...
+
+// Run the Vanilla Brochure Theme
+@include vanilla-brochure-theme;
+
+// Fixes
+
+// .p-strip--light is appearing with a transparent background
+.p-strip--light {
+  background-color: #f7f7f7;
```
Comment (anthonydillon) [inline _sass/main.scss]: Please use the vanilla color.

File: index.html [hunk 4]
```diff
@@ -2,295 +2,256 @@
...
-            </div>
-
-            <h1 class="hero-row__heading">The standard for customising cloud instances</h1>
-        </div>
-
-        <div class="six-col">
-            <a href="https://www.youtube.com/watch?v=-zL3BdbKyGY" class="hero__cta button--primary mediabox">
-                Watch tutorial
-            </a>
+<section class="p-strip p-strip--accent">
```
Comment (anthonydillon) [inline index.html]: Use the class `is-dark`

File: index.html [hunk 5]
```diff
@@ -2,295 +2,256 @@
...
+  <div class="row">
+    <div class="col-12 suffix-4">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/4fd55d51-picto-corepower-orange.svg">
+          <h3 class="p-heading-icon__title" style="color:#fff">cloud-init</h3>
```
Comment (anthonydillon) [inline index.html]: Remove inline styles after adding `is-dark` above.

File: index.html [hunk 6]
```diff
@@ -2,295 +2,256 @@
...
         </div>
+      </div>
+      <h1>The standard for customising cloud instances</h1>
+      <a class="p-link--external p-button--brand" href="https://www.youtube.com/watch?v=-zL3BdbKyGY">Watch tutorial</a>
```
Comment (anthonydillon) [inline index.html]: The external class should be on a span inside the link.

File: index.html [hunk 7]
```diff
@@ -2,295 +2,256 @@
...
+<section class="p-strip">
+  <div class="row">
+    <div class="col-4 suffix-1">
+      <h2>About</h2>
+      <p>cloud-init is developed and released as free software under both the <a href="https://www.gnu.org/licenses/gpl-3.0.en.html">GPLv3 open source
+      license</a> and <a class="p-link--external" href="https://www.apache.org/licenses/LICENSE-2.0"> Apache License version 2.0</a>. It was originally designed for the <a class="p-link--external" href="https://www.ubuntu.com/">Ubuntu</a> distribution of Linux in Amazon EC2, but is now supported on many Linux and UNIX distributions in every major cloud.</p>
+    </div>
+    <div class="col-7">
+      <div class="row">
+        <div class="col-7">
```
Comment (anthonydillon) [inline index.html]: Consider using media-object

**Comment by yaili** [comment]
Design review:
- [ ] The button in hero should have white external icon, and the content should be centred in the button - is this possible in Vanilla?
- [ ] If we make the colour brand orange, will that change link colours, and tick lists?
- [ ] Can we put the text in "Works with many popular operating systems" in a wider container so the titles doesn't wrap in L screens?
- [ ] Can the pattern in "Works with many popular operating systems" restrict the height size of the logos? Like here https://www.ubuntu.com/cloud/public-cloud or https://www.ubuntu.com/internet-of-things
- [ ] We're going to add a circular avatar pattern to Vanilla soon(ish) since it's used across sites, how difficult/bad would it be to add it custom for now here in the team section?
- [ ] The logos in "Works with many popular operating systems" are too large in S screen, can this be fixed? - in "Source code" they look better
- [ ] Add border top and bottom in "USED ACROSS THE PUBLIC CLOUD"

Thanks!

**Comment by anthonydillon** [comment]
Closing this PR as its superseded by https://github.com/canonical-websites/cloud-init.io/pull/63.

**Pull request closed without merging**